lib/PuppeteerSharp/Launcher.cs

```diff
@@ -168,7 +168,29 @@ public Launcher()
             }
 
         }
-        
+
+        /// <summary>
+        /// This methods attaches Puppeteer to an existing Chromium instance.
```

Comment:
maybe removed the words `This methods`?

Reply:
I copy/pasted from Puppeteer.js! I'll remove it and start with "Attaches" directly as you suggest in another comment!